external/fv3fit/fv3fit/reservoir/domain.py

```diff
@@ -65,16 +65,14 @@ def __init__(
         self._partitioner = pace.util.TilePartitioner(subdomain_layout)
 
         # dimensions of rank data without the halo points. Useful for slice calculation.
-        self._rank_extent_without_overlap = self._get_rank_extent_without_overlap(
+        self.rank_extent_without_overlap = self._get_rank_extent_without_overlap(
```

Comment:
Made this a public attribute since it is useful info for other functions to grab